projects/addon-doc/src/components/navigation/navigation.component.ts

```diff
@@ -54,6 +54,8 @@ export class TuiDocNavigationComponent {
     );
 
     readonly mode$: Observable<TuiBrightness> = this.mode.change$.pipe(
+        // startWith(null) is not deprecated despite the notice,
+        // but IDE detects a wrong function signature, which is deprecated
```

Comment:
I am not sure that we need to add comment here.

This bug will be fixed in future, but this comment become some black-box unclear comment)

Reply:
fixed